### PR TITLE
Add ETF symbols to symbols database and best supply/demand data to real-time ticker

### PR DESCRIPTION
**Pull request by hhdri**
I added 47 new symbols (symbols from ETF group) into pytse_clinet/data/symbols_name.json and also replaced Arabic characters 'ي' and 'ك' with their Farsi alternatives for these 47 new symbols.

I found the best supply/demand prices useful (I needed them) so I added them to real-time ticker.

@Glyphack please review and let me know what you think!

### CODE REVIEW

**Comment by Glyphack** [comment]
Thanks for your contribution to pytse-client. I appreciate it. I'll review the symbols you added and merge it as soon as I can.
Also I think you accidentally closed this so I reopen it.

**Comment by hhdri** [comment]
I set a Cron job on a server to capture the response of the real-time data for 7 symbols every 10 minutes. I'll run it for a day or two. After that, I make a zip file from them and upload it somewhere public. These response samples will be useful for debugging the code.

**Comment by hhdri** [comment]
Done sampling responses. It is downloadable at [this link](http://198.143.181.205:8000/samples.zip).

**Comment by Glyphack** [comment]
I can't access it.

**Comment by hhdri** [comment]
> I can't access it.

I fixed the issue. The link is accessible now.

File: pytse_client/ticker.py [hunk 1]
```diff
@@ -9,7 +9,12 @@
 
 RealtimeTickerInfo = collections.namedtuple(
```
Comment (Glyphack) [inline pytse_client/ticker.py:10]: این قابلیت های جدیدی که اضافه شدن به داک هم اضافه بشه واسه استفاده

File: pytse_client/data/symbols_name.json [hunk 2]
```diff
@@ -581,5 +581,51 @@
...
+  "افران": "3846143218462419",
+  "کیان": "53251602435454519",
+  "نگین": "10145129193828624",
+  "فیروزا": "10795723506538053",
+  "رویش": "15124889255100138",
+  "آشناتک": "11301273980316268",
+  "ونچر": "57585821705408565",
+  "نوآور": "63888998422355039",
+  "فنابا": "3329167699100936",
+  "تهران1": "20219449694505209",
```
[the remark this answers is not in the document]
Comment (hhdri) [reply pytse_client/data/symbols_name.json:626]: این یکی رو حذف میکنم

File: pytse_client/ticker.py (hunk 1 above)
Comment (hhdri) [reply pytse_client/ticker.py:10]: Okay

**Comment by hhdri** [comment]
I have added 3 commits:

1. delete symbols with '1' at the end of their names
2. edit get_ticker_real_time_info_response so that it search for best supply/demand, only when we need them. It is because the failure rate of fetching supply/demand data is way more than the failure rate of adj_close/last_price. In this way, we end up with less exceptions
3. Add documentation for best supply/demand data

**Comment by hhdri** [comment]
I couldn't add my commits to this pull request (maybe it's because my repo is behind 11 commits). I'll close this one and add a new one.

**Pull request closed without merging**